COPY/etc/httpd/conf.d/manageiq-https-application.conf

```diff
@@ -19,7 +19,7 @@ TransferLog /var/www/miq/vmdb/log/apache/ssl_access.log
 LogLevel warn
 
 SSLEngine on
-SSLProtocol all -SSLv2 -SSLv3
+SSLProtocol all -SSLv2 -SSLv3 -TLSv1 -TLSv1.1
```

Comment:
In the current version of openssl on RHEL8/CentOS8, SSLv2 and SSLv3 can't be used, so I think we can drop them.